### DISCUSSION TITLE
Feature request: keep the selected area position

### DISCUSSION THREAD

**Discussion by Fajle-Rayhan** [flameshot-org/flameshot#2184] [Ideas]
please add this feature
very needy

![image](https://user-images.githubusercontent.com/17389228/147195295-02fe974b-1550-4713-ad3f-5488f6e442b0.png)

I always use lightshot and this feature alwase, it's pretty handy
i can't use flameshot proparly without this feature. 
so please do it.

**Comment by mmahmoudian** [comment]
Isn't this the exact duplicate of #1903 and #1678?

**Reply by Fajle-Rayhan** [reply]
this is what everyone wants, thous who uses Lightshot -- LightShot, chack it out your self

![113565230-f3db3180-960a-11eb-99fb-0e79c4ef3803](https://user-images.githubusercontent.com/17389228/147378552-ea9ab73a-66bd-4f16-bf2d-9a26d66d5a9a.png)

**Reply by mmahmoudian** [reply]
Please always check the issues to see if your bug report or feature request is already posted.

Regarding this feature request, it is clearly a duplicate, so please follow the program on the related issue.
